ansible/roles/operator-pipeline/templates/openshift/tasks/acquire-lock.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: acquire-lease
+spec:
+  params:
+    - name: lease-name
+      type: string
+      description: The name of the resource which will be acquired.
+    - name: timeout
+      type: string
+      default: 90m
+      description: The timeout for the wait period of leased resource.
+  steps:
+    - name: create-lease
+      image: quay.io/operator_testing/operator-test-playbooks:dev-tekton
```

Comment:
Who owns this repository and what's the support level? Isn't there more trusted sources in the Red Hat registry? I really want to avoid breaking our pipeline in production when somebody decides to remove the image or tag from a repository that we don't own. 

Also please use manifest digests instead of tags.

Reply:
This looks like a good official replacement: https://catalog.redhat.com/software/containers/openshift-pipelines/pipelines-cli-tkn-rhel8/60d062834924e14519331697 Please verify if it has all the required packages that you need.

Reply:
I have verified that the above red hat registry doesn't have required packages and that is why I figured out to use our own operator-pipeline-images with a bunch of changes in commands for the task steps. Can you please review the changes? 

PS: I have tested all of the changes on the operator release pipeline for my local system and it is working fine.

cc @Allda

Reply:
Can you please update the image reference to the one that you suggested in the previous comment? Are you referring to this one? quay.io/redhat-isv/operator-pipelines-images ?